Detectors/TPC/workflow/src/ClustererSpec.cxx

```diff
@@ -36,34 +38,93 @@ using MCLabelContainer = o2::dataformats::MCTruthContainer<o2::MCCompLabel>;
 
 /// create a processor spec
 /// runs the TPC HwClusterer in a DPL process with digits and mc as input
-DataProcessorSpec getClustererSpec()
+DataProcessorSpec getClustererSpec(bool sendMC)
 {
-  auto initFunction = [](InitContext& ic) {
-    auto clusterArray = std::make_shared<std::vector<o2::TPC::Cluster>>();
-    auto mctruthArray = std::make_shared<MCLabelContainer>();
-    auto clusterer = std::make_shared<o2::TPC::HwClusterer>(clusterArray.get(), 0, mctruthArray.get()); // correct sector needs to be set!!
+  constexpr static size_t NSectors = o2::TPC::Sector::MAXSECTOR;
+  struct ProcessAttributes {
+    std::unique_ptr<std::vector<o2::TPC::Cluster>> clusterArray;
```

Comment:
Why a `unique_ptr` to a `std::vector`? `vector`s are already moveable and if the goal is to be able to move the contents without the copy, that should already be possible, no?